test/evergreen.yml

```diff
@@ -38,29 +38,17 @@ functions:
       remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${dependent_task|compile}_${build_id}.tgz
       bucket: build_external
       extract_to: wiredtiger
-  "fetch artifacts from little-endian" :
+  "fetch artifacts from endian" :
```

Comment:
"endian" --> "endian format"

Reply:
Renamed in 070e926bed58849668f694396f32164b06700ba8